gradio/queueing.py

```diff
@@ -144,15 +145,16 @@ def _resolve_concurrency_limit(self, default_concurrency_limit):
 
     async def push(
         self, body: PredictBody, request: fastapi.Request, username: str | None
-    ):
+    ) -> tuple[bool, str]:
```

Comment:
This might have broken `max_size`. 

I ran this demo and submitted jobs on 3 separate tabs:

```py
import time
import gradio as gr

def greet(name):
    time.sleep(20)
    return "Hello " + name + "!"

demo = gr.Interface(fn=greet, inputs="text", outputs="text").queue(max_size=1)
    
if __name__ == "__main__":
    demo.launch(show_api=False)
```

On the final tab, I see this error modal (instead of one telling me that the queue is full):

<img width="788" alt="image" src="https://github.com/gradio-app/gradio/assets/1778297/d261d7c8-cbaf-4ca7-beca-d29c816eeb17">

Reply:
@abidlabs Did you build the frontend before running?

Reply:
I did not in fact :) but after rebuilding it, I still don't see any modal saying that the queue is full. Instead, on the 3rd tab, I just see a job that never enters the queue:

<img width="1315" alt="image" src="https://github.com/gradio-app/gradio/assets/1778297/143944aa-49e7-4538-97ba-856e1591185a">